client-sdks/client-js/src/resources/agent.ts

```diff
@@ -126,6 +127,7 @@ export class Agent extends BaseResource {
         params.experimental_output instanceof ZodSchema
           ? zodToJsonSchema(params.experimental_output)
           : params.experimental_output,
+      runtimeContext: params.runtimeContext ? stringify(Array.from(params.runtimeContext.entries())) : undefined,
```

Comment:
Make this a record instead, why do you need to stringify?

```suggestion
      runtimeContext: params.runtimeContext ? stringify(Object.fromEntries(params.runtimeContext.entries())) : undefined,
```

Reply:
Yeah this is better, I think we dont need to stringify again, we can just send the object